Pod/Classes/BONSpecial.m

```diff
@@ -1,6 +1,6 @@
 //
 //  BONSpecial.m
-//  BonMot
+//  Pods
```

Comment:
#req please fix this in the generator script so it doesn't get reverted the next time someone runs it.